src/milia/api/merged_dataset.cljc

```diff
@@ -0,0 +1,30 @@
+(ns milia.api.merged-dataset
+  (:refer-clojure :exclude [get])
+  (:require [milia.api.http :refer [parse-http]]
+            [milia.utils.remote :refer [make-url]]))
+
+(defn get
+  "Show dataset metadata."
+  [merged-dataset-id & {:keys [no-cache?]}]
+  (let [url (make-url "merged-datasets" (str merged-dataset-id ".json"))]
+    (parse-http :get url :no-cache? no-cache?)))
+
+(defn ^:export data
```

Comment:
Hey I just have a small issue with the formatting and I think the code that builds the dataset suffix is a little too complicated.
I think `#?@(:cljs [:or {:format "json"}])` is really cool. I didn't know that could be done 😄.

What do you think about:

```
(defn ^:export data
  "Return the data associated with a dataset."
  [merged-dataset-id & {:keys [format
                               raw?
                               must-revalidate?
                               accept-header
                               query-params
                               data-id]
                        #?@(:cljs [:or {:format "json"}])}]
  (let [dataset-suffix (when format (str data-id "." format))
        url            (make-url "merged-datasets"
                                 merged-dataset-id
                                 "data"
                                 dataset-suffix)
        options        {:query-params query-params}]
    (parse-http :get              url
                :http-options     options
                :raw-response?    raw?
                :must-revalidate? must-revalidate?
                :accept-header    accept-header)))
```